### PR TITLE
Aggregate lunch_count to agg_child_health

### PR DESCRIPTION
**Pull request by Rohit25negi**
This is to count the number of children given lunch for >=21 a month.

spec: https://docs.google.com/spreadsheets/d/1m6QplhblN7HbOULgUiKqv78At-W5d9RF3u7DYuHosn8/edit#gid=0&range=B21

@calellowitz @emord

### CODE REVIEW

**Comment by Rohit25negi** [comment]
its dependency https://github.com/dimagi/commcare-hq/pull/23154 is deployed. this is ready to merge. But please just review, donn't merge because It involves db migration which might lead to conflicts which are not caught if they are raised after build is done. I will merge only after ensuring no conflict in the migration.

@calellowitz @emord

**Review by calellowitz — APPROVED** [review]
Check the migration for conflicts before merging but otherwise looks good.

**Pull request merged**
